### DISCUSSION TITLE
Where is the liquid documentation of the XML-JSON component?

### DISCUSSION THREAD

**Discussion by stijnmoreels** [invictus-integration/docs-ifa#236] [Q&A]
The XML-JSON component included in Invictus exposes a liquid transform function, but no documentation is available on the documentation site.
Furthermore, not all liquid transformations are currently supported (like: `{{person.firstName | Append: ' ' | Append: person.lastName}}`), which makes it rather hard to use this component.

**Comment by pim-simons** [comment]
Did some searching and from [this thread](https://teams.microsoft.com/l/message/19:94dc05d412d84c16858094ff79af241d@thread.skype/1638473738799?tenantId=7517bc42-bcf8-4916-a677-b5753051f846&groupId=f2ebbeb0-4e8b-4764-9835-98011ae154e9&parentMessageId=1638473738799&teamName=Codit%20Software&channelName=Invictus%20for%20Azure&createdTime=1638473738799) it does not seem that there is any 😞 

I have not used the liquid mapping component so don't have any experience on it, maybe Steve Cleynen can help you out since he [indicated in that thread](https://teams.microsoft.com/l/message/19:94dc05d412d84c16858094ff79af241d@thread.skype/1656067739071?tenantId=7517bc42-bcf8-4916-a677-b5753051f846&groupId=f2ebbeb0-4e8b-4764-9835-98011ae154e9&parentMessageId=1638473738799&teamName=Codit%20Software&channelName=Invictus%20for%20Azure&createdTime=1656067739071) that he might be using it.
